msbuild/Xamarin.Shared/Xamarin.Shared.props

```diff
@@ -210,6 +210,8 @@ Copyright (C) 2020 Microsoft. All rights reserved.
 		<EnableOnDemandResources Condition="'$(_PlatformName)' == 'macOS' And '$(EnableOnDemandResources)' == ''">false</EnableOnDemandResources>
 		<EnableOnDemandResources Condition="'$(_PlatformName)' != 'macOS' And '$(EnableOnDemandResources)' == ''">true</EnableOnDemandResources>
 
+		<UseNativeHttpHandler Condition="'$(_PlatformName)' != 'macOS' And '$(UseNativeHttpHandler)' == ''">true</UseNativeHttpHandler>
```

Comment:
I think this should go with the rest of the runtime feature switches here: https://github.com/xamarin/xamarin-macios/blob/e84984a88330b75048df4424bf95bcb324f83fe2/dotnet/targets/Xamarin.Shared.Sdk.targets#L120-L130

Reply:
Thanks!